### PR TITLE
Fix Column class bug

### LINKED ISSUE #307: Incremental append appears broken with latest version
### Describe the bug
The incremental append appears to have been broken with latest changes

### Steps To Reproduce
1/ Launch a table with following config

{{
    config(
        materialized='incremental',
        incremental_strategy='append'
    )
}}
SELECT a, b
FROM db.table_source

2/ Add a new line to your table_source, relaunch model.
You should have the following error:

insert into table dbt_skeleton_project.glue_incremental_append
select  from dbt_skeleton_project.glue_incremental_append_tmp
ParseException: 
  Syntax error at or near '.'(line 4, pos 37)

Turns out there's no columns after the select.
The bug was introduced with 1.7.1, it is working fine in 1.7.0

### Adapter version
```
dbt-glue==1.7.1
```

### PR DESCRIPTION
**Pull request by sanga8**
resolves #307

- [X] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [X] I have run this code in development and it appears to resolve the stated issue
- [X] This PR includes tests, or tests are not required/relevant for this PR
- [X] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-glue next" section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sanga8** [comment]
@brianhtn

**Comment by sanga8** [comment]
@armaseg How could we resolve this bug asap ? It's a breaking change for incremental models.

File: dbt/adapters/glue/gluedbapi/connection.py [hunk 1]
```diff
@@ -369,7 +369,6 @@ class SqlWrapper2:
     dfs = {}
     @classmethod
     def execute(cls,sql,output=True):
-        sql = sql.replace('"', '')
```
Comment (sanga8) [inline dbt/adapters/glue/gluedbapi/connection.py:372]: @moomindani What was the purpose of this sql change ? It is currently breaking code running with double quotes

Comment (moomindani) [reply dbt/adapters/glue/gluedbapi/connection.py:372]: This was my bad, and I was preparing for fixing this in the next PR. Thanks for your PR.

Comment (sanga8) [reply dbt/adapters/glue/gluedbapi/connection.py:372]: Thank you for your response! no worries

**Comment by moomindani** [comment]
Currently there is some issue in integ tests (not due to this PR). 
While we are working on the fix, could you please verify that your PR works fine with integ test?
Here's the command.

```
% python3 -m pytest -s tests/functional/adapter/
```

**Comment by sanga8** [comment]
> Currently there is some issue in integ tests (not due to this PR). While we are working on the fix, could you please verify that your PR works fine with integ test? Here's the command.
> 
> ```
> % python3 -m pytest -s tests/functional/adapter/
> ```

Ran the tests,
**1 failed, 13 passed, 1 skipped in 989.12s** 

FAILED tests/functional/adapter/test_snapshot.py::TestSnapshotCheckColsGlue::test_snapshot_check_cols - AssertionError: dbt exit state did not match expected

ParseException: 
  Syntax error at or near '"id"'(line 64, pos 34)
  
   from insertions_source_data as source_data
          left outer join snapshotted_data on snapshotted_data.dbt_unique_key = source_data.dbt_unique_key
          where snapshotted_data.dbt_unique_key is null
             or (
                  snapshotted_data.dbt_unique_key is not null
              and (
                  (snapshotted_data."id" != source_data."id"
  ----------------------------------^^^
    
    
I don't understand yet

**Comment by moomindani** [comment]
Apologizes for delay. We needed to fix CI/CD pipeline first.

Regarding the original issue this PR was going to solve, I merged the other PR using your commit.
https://github.com/aws-samples/dbt-glue/pull/322

For double-quote issue, let me create a separate PR. It will require some fix in test case side.

If you do not have further questions, I am going to close this PR.

**Comment by moomindani** [comment]
Regarding double-quote issue, https://github.com/aws-samples/dbt-glue/pull/327 is available.
Closing this PR.

**Pull request closed without merging**